Framework/Algorithms/src/InterpolatingRebin.cpp

```diff
@@ -200,27 +201,18 @@ void InterpolatingRebin::outputYandEValues(
 *  @throw invalid_argument if any output x-values are outside the range of input
 *x-values
 **/
-void InterpolatingRebin::cubicInterpolation(const HistogramData::BinEdges &xOld,
-                                            const MantidVec &yOld,
-                                            const MantidVec &eOld,
-                                            const HistogramData::BinEdges &xNew,
-                                            MantidVec &yNew,
-                                            MantidVec &eNew) const {
-  // Make sure y and e vectors are of correct sizes
+void InterpolatingRebin::cubicInterpolation(const Histogram &oldHistogram,
```

Comment:
@SimonHeybrock I have a problem, which I figured out where it's happening, but I have no idea why. In the state it's currently on the branch, the `WorkspaceHelpers::makeDistribution(outputW, false);`, [found here](https://github.com/mantidproject/mantid/blob/815866e9d46bb390e78960e6d2677f78408dfa44/Framework/Algorithms/src/InterpolatingRebin.cpp#L108) changes the `outputW` values to the ones seen on the test, however after refactoring([refactored code here](http://pastebin.com/115Wc3VA)) that line doesn't seems to change anything in the `outputW` and thus the tests fail. Any ideas why that's happening?